packages/components/src/toggle-group-control/toggle-group-control/as-radio-group.tsx

```diff
@@ -78,22 +78,22 @@ function UnforwardedToggleGroupControlAsRadioGroup(
 				size,
 				value: selectedValue,
 				setValue,
-			} ) as ToggleGroupControlContextProps,
+			} as ToggleGroupControlContextProps ),
```

Comment:
Not sure why, but my IDE auto-formatted the code this way 🤷

Reply:
Not sure why, but my IDE today changed its mind, we're back to the original formatting